src/alembic/README.md

```diff
@@ -0,0 +1,22 @@
+[comment]: <> (TODO write migration guide/contributing with docker-compose commands)
```

Comment:
Can be removed

Reply:
This TODO comment reminds to write a more accurate guide for the database migration, i.e using `PORT=8002 docker-compose exec web <command>` once the process is validated & tested

Reply:
Good idea, let's add the proper instruction in this PR then

Reply:
What do you think ? I added a link from CONTRIBUTING.md to the alembic/README.md